packages/react-components/src/components/MessageThread.tsx

```diff
@@ -868,8 +882,20 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
   const localeStrings = useLocale().strings.messageThread;
   const strings = useMemo(() => ({ ...localeStrings, ...props.strings }), [localeStrings, props.strings]);
 
-  const defaultStatusRenderer: (status: MessageStatus) => JSX.Element = useCallback(
-    (status: MessageStatus) => <MessageStatusIndicator status={status} />,
+  const defaultStatusRenderer: (
+    status: MessageStatus,
+    messageThreadReadCount: number,
+    messageThreadParticipantCount: number
+  ) => JSX.Element = useCallback(
+    (status: MessageStatus, messageThreadReadCount: number, messageThreadParticipantCount: number) => (
+      <MessageStatusIndicator
+        status={status}
+        messageThreadReadCount={messageThreadReadCount}
+        // if messageThreadParticipantCount is less than or equal to 0, we have a bug, set it to 0
+        // -1 because participant count does not include myself
+        participantCountNotIncludingSelf={messageThreadParticipantCount > 0 ? messageThreadParticipantCount - 1 : 0}
```

Comment:
This logic could be covered in packages/react-components/src/components/MessageStatusIndicator.tsx
Line 118:
 participantCountNotIncludingSelf <= 0 ||